src/odemis/driver/tmcm.py

```diff
@@ -368,6 +376,23 @@ def __init__(self, name, role, port, axes, ustepsize, address=None,
             except HwError as ex:
                 # Probably some old error left-over, no need to worry too much
                 logging.warning(str(ex))
+
+        # Add digital output axes
+        self._do_axes = do_axes or {}
+        self._led_prot_do = led_prot_do or {}
+        for port, vals in self._do_axes.items():
+            ax_name, high_pos, low_pos, dur = vals
+            axes_def[ax_name] = model.Axis(choices={low_pos, high_pos})
+            self._name_to_do_axis[ax_name] = port
+
+        self._leds_on = model.BooleanVA(False)  # State of refswitch
+        # Check state of refswitch on startup, just to be sure
+        for refswitch in self._refswitch:
+            if self.GetIO(2, refswitch):
+                self._leds_on.value = True
```

Comment:
*Maybe* the following is more obvious:
```
self._leds_on = any(self.GetIO(2, rs) for rs in self._refswitch)
if self._leds_on:
   logging.warning(...)
```
BTW, avoid reusing the same variable name as the argument of the function. It's confusing, and could lead to bugs if later in the code someone was relying on `refswitch` to still contain the function argument.